public/index.html

```diff
@@ -12,5 +12,39 @@
     <div id="app"></div>
     <script src="build/vendor.js"></script>
     <script src="build/app.js"></script>
+    <script>
+    if ('serviceWorker' in navigator) {
+      navigator.serviceWorker.register('./service-worker.js', {
+        scope: './'
+      })
+      .then(function(registration) {
+        if (typeof registration.update === 'function') {
+          registration.update();
```

Comment:
Good questions. I'm following the pattern we use over here: https://github.com/google/web-starter-kit/blob/master/app/scripts/main.js#L36 but after checking with some of my colleagues, it looks like you're right. This would be way too pedantic :) I'll update.